### PR TITLE
Homepage rework + fixed tests

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | based on #10899 
| License         | MIT

image

### CODE REVIEW

**Comment by kulczy** [comment]
@AdamKasp section between "Newsletter" and "About Us" is missing

File: src/Sylius/Behat/Page/Shop/HomePage.php [hunk 1]
```diff
@@ -19,26 +19,17 @@
...
     public function getRouteName(): string
     {
         return 'sylius_shop_homepage';
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getContents()
+    public function getContents(): string
```
Comment (Tomanhez) [inline src/Sylius/Behat/Page/Shop/HomePage.php]:
```suggestion
    public function getContent(): string
```

File: src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig [hunk 2]
```diff
@@ -3,7 +3,12 @@
         <div class="ui inverted divided equal height stackable grid">
             {{ sonata_block_render_event('sylius.shop.layout.before_footer') }}
 
-            <div class="three wide column">
+            <div class="four wide column">
+                <a href="//sylius.com/plus" target="_blank">
```
Comment (Tomanhez) [inline src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig]:
```suggestion
                <a href="https://sylius.com/plus" target="_blank">
```

**Comment by pamil** [comment]
Thank you, Adam! :1st_place_medal:

**Pull request merged**